HACKING.md

```diff
@@ -9,8 +9,8 @@ Git clone).
 ## This file has moved
 
 Please see the content of this file in its new location:
-https://ompi.readthedocs.io/en/latest/developers/
+https://docs.open-mpi.org/en/main/developers/
```

Comment:
Wouldn't we want to point to the v5.0.0 version of these docs in the v5.0.0 branch?  (in case they change on master in the future)?  I thought that's one of the benefits/drawbacks of release versioned docs.

Reply:
yes..this is a good point. I will amend.

Reply:
I think it is good now.

Reply:
Hm, I guess there are other main references. Let me clean those up.

Reply:
yes, quite a few... Thanks.